garage/tf/core/mlp2.py

```diff
@@ -0,0 +1,80 @@
+"""MLP model in TensorFlow using tf.keras.models.Model."""
+import tensorflow as tf
+from tensorflow.keras.layers import BatchNormalization
+from tensorflow.keras.layers import Dense
+from tensorflow.keras.models import Model
+
+
+def mlp2(input_var,
```

Comment:
It would be a model but whether we want it to be stored in a class or returned by a function?

Reply:
it is a stylistic preference. 

`mlp2()` is technically a factory function, since it returns an instance of a class.

I think it is probably more straightforward to make it a class. That makes it obvious to the user that they are constructing an object and not just calling a function.

Reply:
Ok. I think that would become the `PickableMLP` inside `test_model_autopickable.py`